jetty-annotations/src/test/java/org/eclipse/jetty/annotations/TestAnnotationParser.java

```diff
@@ -71,6 +74,36 @@ public void handle(ClassInfo info, String annotation)
             foundClasses.add(info.getClassName());
         }
     }
+    
+    
+    public static class DuplicateClassScanHandler extends AnnotationParser.AbstractHandler
+    {
+        private Map<String, List<String>> _classMap = new ConcurrentHashMap();
+
+        /** 
+         * @see org.eclipse.jetty.annotations.AnnotationParser.AbstractHandler#handle(org.eclipse.jetty.annotations.AnnotationParser.ClassInfo, java.lang.String)
```

Comment:
Remove this javadoc, it's boilerplate that is not necessary (IDEs will present the javadocs of the super method).

Reply:
Well, my IDE inserted the comment :)  I've deleted it.